source/bson-binary-vector/tests/packed_bit.json

```
@@ -14,30 +14,39 @@
     {
       "description": "Simple Vector PACKED_BIT",
       "valid": true,
-      "vector": [127, 7],
+      "vector": [127, 8],
```

```suggestion
      "vector": [127, 7],
```

Suggest reverting changes to the `Simple Vector PACKED_BIT` test. Since the padding is 0, I expect this test does not need to be updated.

---

Makes sense. I reverted thi.s